### PR TITLE
Update rescale.py

### PR DESCRIPTION
**Pull request by prateekiiest**
issue #2020 done. 
Added some back end  back slashes (``) , so that it doesn't appear to be as broken as in the documentation.

ping @Cadair .

But  I can't find the documentation .rst file for this sunpy.image under the doc/source dir. Its only in the comments, not in an .rst file. Does it ensure the sphinx documentation will be regenerated automatically?

### CODE REVIEW

**Comment by Cadair** [comment]
hey @prateekiiest You should follow this section of the rst primer on embedding source code (backticks is just for verbatim text). http://www.sphinx-doc.org/en/stable/rest.html#source-code

**Comment by Cadair** [comment]
and yes, the API docs are auto generated by sphinx from the docsstrings in the file

File: sunpy/image/rescale.py [hunk 1]
```diff
@@ -163,34 +164,34 @@ def reshape_image_to_4d_superpixel(img, dimensions, offset):
...
-    array
+    array::
 
-    `x = np.array([[0, 0, 0, 1, 1, 1],
-                  [0, 0, 1, 1, 0, 0],
-                  [1, 1, 0, 0, 1, 1],
-                  [0, 0, 0, 0, 1, 1],
-                  [1, 0, 1, 0, 1, 1],
-                  [0, 0, 1, 0, 0, 0]])`
+        `x = np.array([[0, 0, 0, 1, 1, 1],
```
Comment (Cadair) [inline sunpy/image/rescale.py]: you can remove the backticks now ;)

Comment (prateekiiest) [inline sunpy/image/rescale.py]: done

**Pull request merged**